site/write_ups/2022/MetaphorsBlogPost

```diff
@@ -0,0 +1,76 @@
+
+## Conclusion: How Can We All Clean the Oil Spill?
+
+Metaphors are ubiquitous in describing data, but are we just muddying the linguistic and data-ideological waters with our current data metaphors? The answer is, in general, yes. In accordance with the profit-incentivised culture of business, which dominates data science, data scientists, managers, journalists, and others seek to promote the economic value and objective nature of data while, as Hwang and Levy (2015) point out, “people are nowhere to be found” in these data metaphors; the role of humans and ethical concerns regarding data are missing.
+
+What can we do about the oil spill of data metaphors? In keeping with projects like Better Images of AI (2022) that aim to improve representations of data science, I urge data professionals who care about data ethics to create new data metaphors and actively re-interpret existing data metaphors [Note 2].
+
+Devising new metaphors allows for altogether different ways of conceptualising data that may not be allowed by dominant metaphors. In the context of the ‘data as inherent and objective source of knowledge’ data ideology, Benjamin Garfield (2021) offers three alternatives for replacing the term ‘data collection’ that help mitigate the backgrounding of labour and human subjectivity: data creation, data curation, and compiling data. For metaphors falling under ‘data as unprocessed economic potential/resource’, Sara Watson (2015) argues that embodied metaphors can help ‘bring big data back down to a human scale and ground data in lived experience’, helping counter the distancing of data from the humans who created it. To that end, she recommends metaphors from the quantified-self community, such as data is a body (e.g., ‘data is a foot/fingerprint’, ‘data double’). 
+
+In contrast to metaphor creation, reinterpretation provides an avenue for subverting rather than replacing metaphors such as ‘data is the new oil’ that reinforce dominant data ideologies. Instead of solely focussing on the financial profits gained from data being treated as a resource, one can explore alternative entailments of the metaphor to highlight aspects of data usually backgrounded such as environmental externalities; just like extracting oil, processing data can result in significant environmental consequences (see [Note 3] for an example of how you could re-interpret ‘raw data’.) The benefit of reinterpretation as opposed to metaphor creation is that one can more easily build a bridge between existing data ideologies and new data ideologies by simultaneously activating familiar entailments (such as, data has economic value) and new entailments (data processing has environmental costs), helping establish dialogue between people with different perspectives. 
```

Comment:
```suggestion
In contrast to metaphor creation, reinterpretation provides an avenue for subverting rather than replacing metaphors such as ‘data is the new oil’ that reinforce dominant data ideologies. 
Instead of solely focussing on the financial profits gained from data being treated as a resource, one can explore alternative entailments of the metaphor to highlight aspects of data usually backgrounded such as environmental externalities; just like extracting oil, processing data can result in significant environmental consequences (see [Note 3] for an example of how you could re-interpret ‘raw data’.) 
The benefit of reinterpretation as opposed to metaphor creation is that one can more easily build a bridge between existing data ideologies and new data ideologies by simultaneously activating familiar entailments (such as, data has economic value) and new entailments (data processing has environmental costs), helping establish dialogue between people with different perspectives. 
```